src/jquery.mockjax.js

```diff
@@ -728,7 +744,9 @@
 		return i;
 	};
 	$.mockjax.clear = function(i) {
-		if ( i || i === 0 ) {
+		if ( typeof i === 'string' ) {
```

Comment:
If we're going to all matching on the `url` property then we should probably cover all possible values, which includes `RegExp` objects as well. Shouldn't be too much extra effort, I think.